metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "puppetlabs-nodejs",
+  "name": "puppet-nodejs",
   "version": "0.8.0",
```

Comment:
@juniorsysadmin yes, but maybe after merging this PR, juste before releasing a new version